SC_EL not building properly on ubuntu 18.04

## Environment

* SuperCollider version: Source Branch: develop
* Operating system: Ubuntu 18.04
* Other details (Qt version, audio driver, etc.): Emacs and suggested packages were installed with apt, emacs loads, but hasn't been thoroughly tested.

```
emacs/bionic,bionic,now 47.0 all [installed]
emacs25/bionic,now 25.2+1-6 amd64 [installed,automatic]
emacs25-bin-common/bionic,now 25.2+1-6 amd64 [installed,automatic]
emacs25-common/bionic,bionic,now 25.2+1-6 all [installed,automatic]
emacs25-common-non-dfsg/bionic,bionic,now 25.2+1-1 all [installed,automatic]
emacs25-el/bionic,bionic,now 25.2+1-6 all [installed,automatic]
emacsen-common/bionic,bionic,now 2.0.8 all [installed,automatic]
```

## Steps to reproduce

When running make on develop branch on Ubuntu 18.04 with CMAKE flags:
  
`CMAKE_BUILD_TYPE=Release NATIVE=ON SC_EL=ON`


## Expected vs. actual behavior
**make crashes as follows:**

```
[ 93%] Built target SuperCollider
[ 93%] Built target sclang.elc
[ 93%] Built target sclang-interp.elc
[ 93%] Built target sclang-language.elc
[ 93%] Built target sclang-document.elc
[ 94%] Built target sclang-keys.elc
[ 94%] Creating byte-compiled Emacs lisp /home/jms/git/SuperCollider/build/editors/sc-el/el/sclang-help.elc
Loading 00debian-vars...
Loading /etc/emacs/site-start.d/50autoconf.el (source)...
Loading /etc/emacs/site-start.d/50cmake-data.el (source)...
Loading /etc/emacs/site-start.d/50dictionaries-common.el (source)...
Loading debian-ispell...
Loading /var/cache/dictionaries-common/emacsen-ispell-default.el (source)...
Loading /var/cache/dictionaries-common/emacsen-ispell-dicts.el (source)...

In toplevel form:
sclang-help.el:86:7:Warning: assignment to free variable ‘w3m-use-filter’
sclang-help.el:89:18:Warning: reference to free variable ‘w3m-filter-rules’
sclang-help.el:89:18:Warning: assignment to free variable ‘w3m-filter-rules’
sclang-help.el:261:1:Error: Symbol’s function definition is void: gensym
editors/sc-el/el/CMakeFiles/sclang-help.elc.dir/build.make:60: recipe for target 'editors/sc-el/el/sclang-help.elc' failed
make[2]: *** [editors/sc-el/el/sclang-help.elc] Error 1
CMakeFiles/Makefile2:4335: recipe for target 'editors/sc-el/el/CMakeFiles/sclang-help.elc.dir/all' failed
make[1]: *** [editors/sc-el/el/CMakeFiles/sclang-help.elc.dir/all] Error 2
Makefile:162: recipe for target 'all' failed
make: *** [all] Error 2
```

Works fine for me if i set SC_EL=OFF. I don't need a fix, but wanted to let you know, since SC_EL=ON is the default.

--- a/el/sclang-help.el
+++ b/el/sclang-help.el
@@ -192,9 +192,9 @@
   `(push (cons ,font-id (intern ,font-name)) (sclang-rtf-state-font-table ,state)))
 
 (defmacro sclang-rtf-state-apply (state)
-  (let ((pos (gensym))
-	(font (gensym))
-	(face (gensym)))
+  (let ((pos (cl-gensym))
+	(font (cl-gensym))
+	(face (cl-gensym)))
     `(with-current-buffer (sclang-rtf-state-output ,state)
        (let ((,pos (or (sclang-rtf-state-pos ,state) (point-min)))
 	     (,font (cdr (assq
@@ -216,7 +216,7 @@
      (setf (sclang-rtf-state-font ,state) ,font)))
 
 (defmacro sclang-rtf-state-push-face (state face)
-  (let ((list (gensym)))
+  (let ((list (cl-gensym)))
     `(let ((,list (sclang-rtf-state-face state)))
        (sclang-rtf-state-apply ,state)
        (unless (memq ,face ,list)
@@ -224,7 +224,7 @@
 	       (append ,list (list ,face)))))))
 
 (defmacro sclang-rtf-state-pop-face (state face)
-  (let ((list (gensym)))
+  (let ((list (cl-gensym)))
     `(let* ((,list (sclang-rtf-state-face ,state)))
        (sclang-rtf-state-apply ,state)
        (setf (sclang-rtf-state-face ,state) (delq ,face ,list)))))
@@ -326,9 +326,9 @@
   (define-key map "\C-c\C-v" 'sclang-edit-help-file))
 
 (defmacro sclang-help-mode-limit-point-to-code (&rest body)
-  (let ((min (gensym))
-	(max (gensym))
-	(res (gensym)))
+  (let ((min (cl-gensym))
+	(max (cl-gensym))
+	(res (cl-gensym)))
     `(if (and (sclang-code-p (point))
 	      (not (or (bobp) (eobp)))
 	      (sclang-code-p (1- (point)))

